dask_expr/io/io.py

```diff
@@ -39,19 +48,109 @@ def _layer(self):
 
 
 class BlockwiseIO(Blockwise, IO):
-    pass
+    _absorb_projections = False
```

Comment:
Note that it probably makes sense to work with a `BlockwiseIO` subclass here (instead of adding `_absorb_projections`). However, I used the attribute to explore and haven't bothered to modify the approach yet. I'll be happy to revise.

Reply:
I think that's fine for now